.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["env", { "modules": false }], "react"],
+  "presets": [["env", { "modules": false }], "stage-3", "react"],
```

Comment:
Let's roll stage-3. I don't think the object spread syntax is going to change, and dang it's handy.